src/CommentsList.jsx

```diff
@@ -1,5 +1,32 @@
-const CommentsList = () =>{
+import { useState, useEffect } from "react"
+import { fetchCommentsByArticle } from "./api"
 
+const CommentsList = ({article}) =>{
+    const [comments, setComments] = useState([])
+ 
+    useEffect(() => {
+        fetchCommentsByArticle(article.article_id).then((data)=>{
+            
+            setComments(data)
+        })
+    }, [])
+
+    return(
+        <div>
+            {comments.map((comment) => {
```

Comment:
please add some conditional logic to give user feedback based on if commnets are loading, or if article has no comments